### PR TITLE
fix: correct message and translation key

### LINKED ISSUE #15656: Use correct translation key
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Use correct translation key

### Steps to Reproduce

1. (for example) Went to ...
2. Clicked on...
3. ...

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

- What's happening right now that is different from what is expected

### Expected Results

- This is an ideal result that the system should get after the tests are performed

### Technical details

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version
- Anything else that you think could be an issue.

### Evidence

- How was this tested? This is quite mandatory in terms of bugs. Providing evidence of your testing with screenshots or/and videos is an amazing way to prove the bug and a troubleshooting chance to find the solution.

### PR DESCRIPTION
**Pull request by RushikeshGandhmal**
## What does this PR do?

- Fixes #15656 
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] N/A- I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Comment by RushikeshGandhmal** [comment]
So here we were using event_type_requres_google_cal which was not present. Correct one is requires_google_calendar.
Based on it I changes the message.

Can we add new message ? with link redirecting to google_calendar to install as it requires.

@PeerRich @hariombalhara

**Comment by RushikeshGandhmal** [comment]
Before: 
before

After:
after

**Review by anikdhabal — COMMENTED** [review]
Hey @RushikeshGandhmal, I think the message is correct. We should change the link to redirect to the Google Calendar in the App Store. Also, please add the missing translation in the locales using this key.

And instead of 'change it here,' we should add 'connect Google Calendar.'

**Review by anikdhabal — CHANGES_REQUESTED** [review]
Left messages, let me know what do you think?

**Comment by RushikeshGandhmal** [comment]
@anikdhabal that's what I though in the first place and asked the confirmation above. I've pushed the code.

The message can we better, to make it easy for user to read and understand @PeerRich @hariombalhara

**Review by anikdhabal — CHANGES_REQUESTED** [review]
Hey, the i18n key was correct previously, no need to change it. And instead of 'Install here,' add 'Connect it here.

File: apps/web/components/eventtype/EventSetupTab.tsx [hunk 1]
```diff
@@ -417,13 +417,11 @@ export const EventSetupTab = (
...
               </div>
-              <Trans i18nKey="event_type_requres_google_cal">
+              <Trans i18nKey="event_type_requires_google_calendar">
                 <p>
                   The “Add to calendar” for this event type needs to be a Google Calendar for Meet to work.
                   Change it{" "}
-                  <Link
-                    href={`${WEBAPP_URL}/event-types/${formMethods.getValues("id")}?tabName=advanced`}
-                    className="underline">
+                  <Link href={`${WEBAPP_URL}/apps/google-calendar`} className="underline">
```
Comment (hariombalhara) [inline apps/web/components/eventtype/EventSetupTab.tsx:424]: Why is this link changed?

Also, could you test with a dummy translation for any non-english language that confirms that it works.

**Review by hariombalhara — DISMISSED** [review]
@RushikeshGandhmal Thanks for the PR 🙏 I left a comment

File: apps/web/components/eventtype/EventSetupTab.tsx (hunk 1 above)
Comment (anikdhabal) [reply apps/web/components/eventtype/EventSetupTab.tsx:424]: In our code, we check if the conferencing app is Google Meet and the `destinationCalendar?.integration !== "google_calendar"`. If it isn't, we show this message. Previously, the link redirected to the event advanced tab, but this is unrelated.

**Review by anikdhabal — COMMENTED** [review]
Hey @RushikeshGandhmal, it’s not fixed yet. Take the bottom i18nkey in this component as a reference. And please test it before the final commit. Also, instead of 'change it here,' use 'Connect it here.'

**Comment by RushikeshGandhmal** [comment]
@anikdhabal please take a look at screenshots before committing. 

Screenshot from 2024-07-04 13-23-38
Screenshot from 2024-07-04 13-23-06
Screenshot from 2024-07-04 13-22-05

Once confirmed I'll add the sentence in all the required languages.

**Comment by anikdhabal** [comment]
> Once confirmed I'll add the sentence in all the required languages.
Yeah this is correct. Commit the changes and you don't need to add the i18n key to every locales. Just add it in /locales/en

**Comment by anikdhabal** [comment]
Great work @RushikeshGandhmal pls commit the changes. 
And You don't need to add the i18n key in every locales. Currently it's correct.

**Comment by RushikeshGandhmal** [comment]
Sure @anikdhabal. Thank you for right direction. Appreciate that.

and yes @hariombalhara for any non-english language it works. But as @anikdhabal said we don't need to add to other locals. It's in english only. Let me know if I can add in other languages also.

Screenshot from 2024-07-04 18-51-06

**Comment by anikdhabal** [comment]
> Sure @anikdhabal. Thank you for right direction. Appreciate that.
> 
> and yes @hariombalhara for any non-english language it works. But as @anikdhabal said we don't need to add to other locals. It's in english only. Let me know, I can add in other languages also.
> 
hey in our project we handle it externally for other language, you don't need to do extra work.

**Comment by RushikeshGandhmal** [comment]
Sure @anikdhabal. Thanks for informing.

**Comment by RushikeshGandhmal** [comment]
@hariombalhara gotcha. Yeah, I've tested working well for all languages.
Example above for Deutsch.


rushikesh.

**Review by anikdhabal — APPROVED** [review]
LGTM

**Pull request merged**